content/en/spec/activitypub.md

```diff
@@ -116,58 +116,58 @@ Links will be kept if the protocol is supported, and converted to text otherwise
+: Used to determine whether status media or text should be hidden by default. See the [Sensitive content](#sensitive) extension section for more information about `as:sensitive`.
 
 inReplyTo
-: Used for threading a status as a reply to another status
+: Used for threading a status as a reply to another status.
 
 published
-: Used as status date
+: Used as status date.
 
 url
-: Used for status permalinks, and also appended to status text for transformed Objects
+: Used for status permalinks, and also appended to status text for transformed Objects.
 
 attributedTo
-: Used to determine the profile which authored the status
+: Used to determine the profile which authored the status.
 
 to/cc
-: Used to determine audience and visibility of a status, in combination with mentions. See [Mentions for adddressing and notifications](#Mention)
+: Used to determine audience and visibility of a status, in combination with mentions. See [Mentions for addressing and notifications](#Mention).
 
 tag
 : Used to mark up mentions and hashtags.
 
 tag[].type
-: Either `Mention`, `Hashtag`, or `Emoji` is currently supported. See the [Hashtag](#Hashtag) and [Custom emoji](#Emoji) extension sections for more information
+: Either `Mention`, `Hashtag`, or `Emoji` is currently supported. See the [Hashtag](#Hashtag) and [Custom emoji](#Emoji) extension sections for more information.
 
 tag[].name
-: The plain-text Webfinger address of a profile Mention (`@user` or `@user@domain`), or the plain-text Hashtag (`#tag`), or the custom Emoji shortcode (`:thounking:`)
+: The plain-text WebFinger address of a profile Mention (`@user` or `@user@domain`), or the plain-text Hashtag (`#tag`), or the custom Emoji shortcode (`:thounking:`).
 
 tag[].href
-: The URL of the actor or tag
+: The URL of the actor or tag.
 
 attachment
 : Used to include attached images, videos, or audio.
 
 attachment[].url
-: Used to fetch the media attachment
+: Used to fetch the media attachment.
 
 attachment[].summary
-: Used as media description
+: Used as media description.
 
 attachment[].blurhash
-: Used to generate a blurred preview image corresponding to the colors used within the image. See [Blurhash](#blurhash) for more details.
+: Used to generate a blurred preview image corresponding to the colours used within the image. See [Blurhash](#blurhash) for more details.
 
 replies
-: A Collection of statuses that are in reply to the current status. Up to 5 replies from the same server will be fetched upon discovery of a remote status, in order to resolve threads more fully. On Mastodon's side, the first page contains self-replies, and additional pages contain replies from other people.
+: A Collection of statuses that are in reply to the current status. Up to five replies from the same server will be fetched upon discovery of a remote status, in order to resolve threads more fully. On Mastodon's side, the first page contains self-replies, and additional pages contain replies from other people.
```

Comment:
the number is more readable when expressed numerically, so this change is less readable